language/stdlib/src/lib.rs

```diff
@@ -165,14 +165,34 @@ fn build_abi(output_path: &str, sources: &[String], dep_path: &str, compiled_scr
     move_prover::run_move_prover_errors_to_stderr(options).unwrap();
 }
 
+fn post_process_add_proptest_derivations(buffer: &[u8]) {
```

Comment:
Can you add a TODO pointing to https://github.com/facebookincubator/serde-reflection/issues/32 ?

Reply:
Will do!